### ISSUE TITLE
There is an extra step "Select subscription" when deploying changes by clicking "Yes" button from the prompt notification

### ISSUE THREAD

**Issue by v-xinda** [microsoft/vscode-azurecontainerapps#683]
**OS:** Win10
**Build Version:** 20240430.1
**Regression:** Not a regression 

**Repro Steps:**
1. Make sure more than one subscription is selected.
2. Execute "Edit Scaling Range..." action for a container app.
3. Click "Yes" button in the prompt notification "Deploy changes now?".
![image](https://github.com/microsoft/vscode-azurecontainerapps/assets/57662131/f4355383-876a-4a77-96b9-8ad211890190)
4. Check whether there is no extra step "Select subscription".

**Expect:**
There is no extra step "Select subscription".

**Actual:**
There is an extra step "Select subscription".
![image](https://github.com/microsoft/vscode-azurecontainerapps/assets/57662131/81957766-0a09-4ed5-8776-12ea33a7a094)

**Comment by MicroFish91** [comment]
Good catch.  When we call the deploy command we should probably attach the existing subscription so that we don't end up re-prompting the user for this info.
